packages/workbox-cache-expiration/src/lib/cache-expiration.js

```diff
@@ -165,6 +166,8 @@ class CacheExpiration {
           // Only return false if all the conditions are met.
           return false;
         }
+      } else {
+        this.expireEntries({cacheName, now});
```

Comment:
If there is no maxAgeSeconds, can we really regard the entry as fresh?

I'm just slightly confused at what this is solving.